app/frontend/src/features/messages/groupchats/CreateGroupChat.tsx

```diff
@@ -0,0 +1,107 @@
+import { BoxProps } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import { Autocomplete } from "@material-ui/lab";
+import React, { useEffect, useState } from "react";
+import { Controller, useForm } from "react-hook-form";
+import Alert from "../../../components/Alert";
+import Button from "../../../components/Button";
+import CircularProgress from "../../../components/CircularProgress";
+import TextField from "../../../components/TextField";
+import { User } from "../../../pb/api_pb";
+import { service } from "../../../service";
+import { useAppDispatch, useTypedSelector } from "../../../store";
+import { fetchUsers, getUsers } from "../../userCache";
+
+const useStyles = makeStyles({ root: {} });
+
+export interface GroupChatListProps extends BoxProps {
+  createGroupChat: (title: string, users: User.AsObject[]) => void;
+}
+
+interface CreateGroupChatFormData {
+  title: string;
+  users: User.AsObject[];
+}
+
+export default function CreateGroupChat({
+  createGroupChat,
+}: GroupChatListProps) {
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState("");
+  const [friendIds, setFriendIds] = React.useState<number[]>([]);
+  const allUsers = useTypedSelector((state) => getUsers(state));
+  const friends = friendIds
+    .map((friendId) => allUsers["" + friendId]?.user)
```

Comment:
```suggestion
    .map((friendId) => allUsers[friendId]?.user)
```
Since it's concat'ing to a blank string, so this is equivalent and simpler?